<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Research - Cheng Lab at UC Davis</title>
  <link rel="stylesheet" href="style.css">
</head>
<body>
  <div class="container">
    <aside class="sidebar">
      <img src="images/ChengLabLogo.png" alt="Cheng Lab Logo" class="sidebar-logo">
      <h1>Cheng Lab</h1>
      <ul class="nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html" class="active">Research</a></li>
        <li><a href="software.html">Software</a></li>
        <li><a href="teaching.html">Teaching</a></li>
      </ul>
    </aside>
    <main class="main-content">
      <header>
        <img src="images/lab_logo.png" alt="Lab Photo" class="lab-photo">
        <div class="lab-info">
          <h2>Cheng Lab at UC Davis</h2>
          <p>Statistical and Quantitative Genetics<br>Department of Animal Science, UC Davis</p>
        </div>
      </header>
      <section class="research-themes">
        <div class="research-vision">
          <h4><strong>Statistical Methods for G2P</strong></h4>
          <img src="images/Research/StatisticalMethods.png" alt="Statistical Methods for G2P" class="research-img" style="width: 400px; float: left; margin-right: 16px; margin-bottom: 8px; border-radius: 6px;">
          <p>Our research primarily focuses on the development of statistical models, theories in quantitative genetics, and the analysis of complex datasets. We have developed statistical methods, computational approaches, and quantitative genetics theories for <a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-020-00533-x" target="_blank">faster genomic prediction</a>, <a href="https://www.genetics.org/content/209/1/89.short" target="_blank">multi-trait analysis addressing pleiotropy</a> and <a href="https://www.g3journal.org/content/early/2020/10/05/g3.120.401618" target="_blank">indirect effects</a>, <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.00362/full" target="_blank">linkage disequilibrium</a>, <a href="https://academic.oup.com/g3journal/article/11/10/jkab228/6318779" target="_blank">integrating mixed models with neural networks to address non-linearity</a> and <a href="https://academic.oup.com/genetics/article-abstract/221/1/iyac034/6536967?redirectedFrom=fulltext">multi-layer biological data</a>, <a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.740340/full" target="_blank">sequencing strategies</a>, among many others. We have also developed software tools to implement these methods. Many of these innovations are incorporated into the JWAS software tool, which has become a pivotal platform for various research projects in both academia and industry.</p>
        </div>

        <div class="research-vision">
          <h4><strong>Molecular Phenotypes in G2P</strong></h4>
          <img src="images/Research/Metabolite.png" alt="Molecular Phenotypes in G2P" class="research-img" style="width: 260px; float: left; margin-right: 16px; margin-bottom: 8px; border-radius: 6px;">
          <p>Advancements in high-throughput sequencing technologies have greatly expanded the amount and diversity of multi-omics data available for animal studies. Our work involves developing statistical methods and tools, as well as performing data analysis on epigenomic, gene expression, and metabolite datasets. For example, we are involved in USDA-NIFA funded projects in partnership with animal genetics industry to collect metabolite data from their breeding populations, aiming to evaluate the role of metabolite data in Genome-to-Phenome (G2P) research. We also engage in projects that integrate public omic datasets, such as those from FAANG and FarmGTEx, into Agricultural Genome-to-Phenome (AG2P) studies.</p>
        </div>

        <div class="research-vision">
          <h4><strong>Public and Confidential Data</strong></h4>
          <img src="images/Research/DataSharing.png" alt="Public and Confidential Data" class="research-img" style="width: 260px; float: left; margin-right: 16px; margin-bottom: 8px; border-radius: 6px;">
          <p>Data sharing and collaboration are increasingly important to enable validation, further research, and joint analysis of multiple datasets. However, these processes are often complicated by limited visibility of public data and confidentiality of private data. I initiated a project to develop platforms and methods for sharing and collaborating on agricultural genome-to-phenome (AG2P) research using both public and confidential data, supported by a USDA-NIFA grant. Our approach includes fostering public data sharing, innovating encrypted data sharing, and enabling collaborative research such as federated and transfer learning. We have developed methods to <a href="https://academic.oup.com/genetics/article/226/3/iyad210/7470728">encrypt genotypes and phenotypes for collaborative analyses</a> and established a Stakeholder Advisory Board to ensure our methods meet the needs of the community.</p>
        </div>
        <div class="research-vision">
          <h4><strong>Machine Learning Beyond Numerical Phenotypes (e.g., Images)</strong></h4>
          <p>Complex phenotypes are sometimes better understood through images or words rather than through numerical values. We have developed machine learning tools specifically designed to analyze these phenotypes within the context of G2P research.</p>
          <ul>
            <li><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/vru.12901" target="_blank">AI-powered Veterinary Radiograph Image Prescreen</a></li>
            <li><a href="https://academic.oup.com/jas/article/100/6/skac147/6576119" target="_blank">VTag: A semi-supervised pipeline for tracking pig activity with a single top-view camera</a></li>
          </ul>
        </div>

        <div class="research-vision">
          <h4><strong>Machine learning for Basic Science</strong></h4>
          <img src="images/Research/ML4BasicScience.jpg" alt="Machine Learning for Basic Science" class="research-img" style="width: 260px; float: left; margin-right: 16px; margin-bottom: 8px; border-radius: 6px;">
          <p>We explore the application of machine learning in fundamental scientific research, with a focus on the genetic and evolutionary bases of complex phenotypes. For example, we developed <a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(23)00187-8" target="_blank">a deep learning-based approach to analyze genomic conservation at the functional level across species</a>, specifically examining functional profiles from humans and pigs. This research highlights the significant role of cross-species functional comparisons in understanding the genetic and evolutionary foundations of complex phenotypes. By bridging deep learning, comparative genomics, and quantitative genetics, we aim to uncover fundamental biological mechanisms and evolutionary patterns.</p>
        </div>

        <div class="research-vision">
          <h4><strong>Applied Quantitative Genetics for Various Species</strong></h4>
          <p>Application of these methods and tools to datasets representing a wide variety of traits across different species, we have applied both known and newly developed methods to analyze datasets from species such as swine, chicken, cattle, trout, maize, walnut, and others. In these analyses, we performed genomic analyses to predict desired traits and infer the underlying mechanisms driving them.</p>
          <ul>
            <li><strong>Trout</strong>
              <ul>
                <li><a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0489-z" target="_blank">Genome-wide association analysis and accuracy of genome-enabled breeding value predictions for resistance to infectious hematopoietic necrosis virus in a commercial rainbow trout breeding population</a></li>
                <li><a href="https://www.frontiersin.org/articles/10.3389/fvets.2020.590048/full" target="_blank">Assessing Accuracy of Genomic Predictions for Resistance to Infectious Hematopoietic Necrosis Virus With Progeny Testing of Selection Candidates in a Commercial Rainbow Trout Breeding Population</a></li>
              </ul>
            </li>
            <li><strong>Cattle</strong>
              <ul>
                <li><a href="https://academic.oup.com/biolreprod/article-abstract/100/4/885/5247713?redirectedFrom=fulltext" target="_blank">In vitro breeding: application of embryonic stem cells to animal production</a></li>
                <li><a href="http://doi.org/10.1371/journal.pone.0200069" target="_blank">Runs of homozygosity in a selected cattle population with extremely inbred bulls: Descriptive and functional analyses revealed highly variable patterns</a></li>
              </ul>
            </li>
            <li><strong>Walnut</strong>
              <ul>
                <li><a href="https://academic.oup.com/jxb/article/71/3/1107/5602662" target="_blank">Combining phenotype, genotype and environment to uncover genetic components underlying water use efficiency in Persian walnut</a></li>
                <li><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231144" target="_blank">Quantitative phenotyping of shell suture strength in walnut (Juglans regia L.) enhances precision for detection of QTL and genome-wide association mapping</a></li>
                <li><a href="https://www.frontiersin.org/articles/10.3389/fpls.2019.01140/full" target="_blank">Deciphering of the Genetic Control of Phenology, Yield, and Pellicle Color in Persian Walnut (Juglans regia L.)</a></li>
              </ul>
            </li>
            <li><strong>Chicken</strong>
              <ul>
                <li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/jbg.12679" target="_blank">Genome-wide association studies for egg quality traits in White Leghorn layers using low-pass sequencing and SNP chip data</a></li>
              </ul>
            </li>
            <li><strong>Others</strong>
            </li>
          </ul>
        </div>

      </section>
    </main>
  </div>
  <footer>
    <p>Cheng Lab, Statistical and Quantitative Genetics, UC Davis</p>
  </footer>
</body>
</html> 